src/test/java/seedu/address/model/person/FamilyTest.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class FamilyTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Family(null));
+    }
+
+    @Test
+    public void constructor_invalidFamily_throwsNumberFormatException() {
+        String invalidFamily = "";
+        assertThrows(NumberFormatException.class, () -> new Family(invalidFamily));
+    }
+
+    @Test
+    public void constructor_invalidFamily_throwsNumberFormatException2() {
+        String invalidFamily = " ";
+        assertThrows(NumberFormatException.class, () -> new Family(invalidFamily));
+    }
+
+    @Test
+    public void constructor_invalidFamily_throwsNumberFormatException3() {
+        String invalidFamily = " family"; //alphabets
+        assertThrows(NumberFormatException.class, () -> new Family(invalidFamily));
+    }
```

Comment:
Nitpick: maybe can standardise the test comments to a new line 
```suggestion
    @Test
    public void constructor_invalidFamily_throwsNumberFormatException3() {
        //invalid input containing alphabets
        String invalidFamily = " family";
        assertThrows(NumberFormatException.class, () -> new Family(invalidFamily));
    }
```

Reply:
Good spot! Committed